ui/src/routes/devices.tsx

```diff
@@ -101,3 +101,5 @@ export default function DevicesRoute() {
     </div>
   );
 }
+
+DevicesRoute.loader = loader;
```

Comment:
Fixing the eslint warning about the `export const loader` requires a change here to follow the same pattern as other routes.